packages/conf-fdkaac/conf-fdkaac.1/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "2.0"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+homepage: "https://github.com/mstorsjo/fdk-aac"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+authors: "fdk-aac dev team"
+license: "Fraunhofer-FDK-AAC-for-Android "
+build: ["pkg-config" "--exists" "fdk-aac"]
+depends: [
+  "conf-pkg-config" {build}
+]
+depexts: [
+  ["libfdk-aac-dev"] {os-family = "debian" | os-family = "ubuntu"}
+  ["fdk-aac-dev"] {os-family = "alpine"}
+  ["fdk-aac-free-devel"] {os-distribution="centos" | os-family = "fedora" | os-family = "suse"}
+  ["fdk-aac"] {os-distribution = "nixos" | os = "freebsd" | os = "macos" & os-distribution = "homebrew"}
+  ["libfdk-aac"] {os-distribution = "arch"}
+]
+synopsis: "Virtual package relying on fdk-aac"
+description:
+  "This package can only install if the fdk-aac is installed on the system."
+flags: conf
```

Comment:
```suggestion
flags: conf
x-ci-accept-failures: ["oraclelinux-8"]
```

The package doesn't seem to be found by the CI on oracle8

Reply:
abour depian need this deb-multimedia-keyring to install libfdk-aac-dev

echo "deb http://www.deb-multimedia.org bullseye main" >> /etc/apt/sources.list
apt-get -y update -oAcquire::AllowInsecureRepositories=true
apt-get -y install deb-multimedia-keyring

test on Debian 11

Reply:
Yes, the `fdk-aac` packages are not always available b/c of the restrictive license. It's actually pretty new that they are available on rpm. 

The packages are also available on debian non-free but it's not enabled by default: https://packages.debian.org/sid/libfdk-aac-dev